### PR TITLE
IMMENSE BUFF TO SECURE BRIEFCASES/WALLSAFE

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
:cl: Cobby
tweak: Nanotrasen has buffed the electronic safety devices found in Secure Briefcases and Wallsafes, making them... well... more secure. In Particular, we added more wires that do absolutely nothing but hinder the use of the multitool, along with removing the ID slot only used by syndicates to emag the safe.
/:cl:
# **TRIGGER WARNING: HIGHLY CONTROVERSIAL**

SAY GOODBYE EMAG USERS! THE AGE OF THE MULTITOOL IS UPON US!

**In Short:** Removes Emag Function [aka Requiring a multitool], and has a lower chance to successfully hack [from 40% to 33%]

It doesn't make sense that something that has no place for an ID can be swiped with an overglorified ID. Git gud and use the multitool. Now when someone puts secure items in the secure safe, people won't laugh at them [i mean they probably still will, but it's ok].

### CODE REVIEW

File: code/game/objects/items/weapons/storage/secure.dm [hunk 1]
```diff
@@ -65,13 +65,10 @@
...
-			emagged = 1
-			src.add_overlay(image('icons/obj/storage.dmi', icon_sparking))
-			sleep(6)
-			src.overlays = null
-			add_overlay(image('icons/obj/storage.dmi', icon_locking))
-			locked = 0
-			user << "<span class='notice'>You short out the lock on [src].</span>"
+			user << "<span class='notice'>You take a second to realize there's no ID slot for you to emag this with.</span>"
+	else
+		if(!emagged)
```
Comment (lordpidey) [inline code/game/objects/items/weapons/storage/secure.dm]: Remove the whole emagged variable, it doesn't do anything now anyway.

**Comment by ShadowDeath6** [comment]
change remove to removing in your cl

also this is pretty good, now cap can put the disk into his secure safe without being called a dumbshit as often

**Comment by lordpidey** [comment]
Noo!  Keep the sassy messages!  They're hilarious!

**Comment by optimumtact** [comment]
code/game/objects/items/weapons/storage/secure.dm:75:error: src.emagged: undefined var
   code/game/objects/items/weapons/storage/secure.dm:77:error: src.emagged: undefined var
   code/game/objects/items/weapons/storage/secure.dm:96:error: src.emagged: undefined var
   code/game/objects/items/weapons/storage/secure.dm:104:error: src.emagged: undefined var

**Comment by ExcessiveUseOfCobblestone** [comment]
CRAWLING IN MY SKIN

**Comment by Cheridan** [comment]
will probably be too strong a deterrent against nuke ops, who generally don't have time to fuck around with tools

**Comment by PKPenguin321** [comment]
yeah this is kind of a pointless nuke op nerf, also is there any actual delay to using a multitool on this because if not probs dont really matter

**Comment by ExcessiveUseOfCobblestone** [comment]
I think you can bypass it all with a c4 though.

**Comment by PKPenguin321** [comment]
can it though? actually test because last i checked C4 cant be planted on storage items, it tries to put the C4 in them

**Comment by ExcessiveUseOfCobblestone** [comment]
yes I just checked. It MAY not let you c4 it when it's unlocked, but uh.... what's the point in that.

**Comment by PKPenguin321** [comment]
it's not a meme for nuke ops though, 99% of them do not carry multitools and getting one when the crew is out to rape you constantly is not as easy as you'd imagine

**Comment by kevinz000** [comment]
i see that clusterbang line underneath
would be lovely if you could fix it and maybe uncomment it.

**Comment by ExcessiveUseOfCobblestone** [comment]
Then prepare better? You already get a toolbox in the ship, and it isn't exactly hard to raid engineering when a majority of ops go near engineering to bomb tcomms. You can even use your emag there on the locker to get a locker-full of multitools [and screwdrivers since you have to open up the wires]. With that said, c4 WILL cuck the safe, so just keep a c4 handy which you should do anyways.

The Elusive Emag being a one-hit-wonder on nearly every item that remotely deals with electricity is silly.

**Comment by ExcessiveUseOfCobblestone** [comment]
ok my brainfart is over, i'm retarded, and now it should compile without the emag var

**Comment by AdamElTablawy** [comment]
there are multitools on the god damn ship i'm fairly certain a nukeop team well armed enough to get past the initial tide and into the cap's office with a captain stupid enough to leave it in the safe can actually multitool it open

not to mention this makes the safe WAY easier to open instead of harder-the secure safe is now stupid easy to open instead of 'have emag or DIE'.

so yes, in favor

**Comment by PKPenguin321** [comment]
there's like 1 multitool on the ship iirc, in the syndie toolbox, that's it

File: code/game/objects/items/weapons/storage/secure.dm [hunk 2]
```diff
@@ -62,17 +61,14 @@
...
-			locked = 0
-			user << "<span class='notice'>You short out the lock on [src].</span>"
-
+/obj/item/weapon/storage/secure/attackby(obj/item/weapon/W, mob/user, params)
+	if(istype(W, /obj/item/weapon/card/emag))
+		if(locked)
+			user << "<span class='notice'>You take a second to realize there's no ID slot for you to emag this with.</span>"
+		else
+			user << "<span class='notice'>You take a second to realize the case is already open, making you look really silly. Hopefully no one saw.</span>"
+	return
```
Comment (RemieRichards) [inline code/game/objects/items/weapons/storage/secure.dm]: trailing return, remove it please.

**Review by RemieRichards — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weapons/storage/secure.dm (hunk 2 above)
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/weapons/storage/secure.dm]: done

**Review by RemieRichards — APPROVED** [review]

**Comment by ExcessiveUseOfCobblestone** [comment]
>there's like 1 multitool on the ship iirc, in the syndie toolbox, that's it

couple that with the grenade, the syndicate beacons, and not to mention their own TC pool, there should be ZERO reason why they can't bust a static object.

**Comment by PKPenguin321** [comment]
honestly i would just remake these things so that they're just like personal lockers. if you have high enough ID access you can open them no matter what, otherwise they're only unlockable by whoever swipes their ID in them first (or by emags/EMPs)

**Comment by Bawhoppen** [comment]
a PR we can all get behind :+1:

**Comment by ExcessiveUseOfCobblestone** [comment]
> instead of taking the emag portion away and keeping the unique portion of the safes, try to rework them entirely so they thematically make sense to use the emag to the point where they're no different from the other storage units outside of aesthetics.

No more memes please.

**Comment by PKPenguin321** [comment]
I don't really understand the necessity of making it emag proofed anyways

>inb4 realism argument

**Comment by Ultimate-Chimera** [comment]
rip memes

**Comment by xxalpha** [comment]
Stop telling whole fucking stories in your changelogs, everyone.

**Comment by ExcessiveUseOfCobblestone** [comment]
It's hip to make roleplay oriented changelogs.

File: code/game/objects/items/weapons/storage/secure.dm [hunk 3]
```diff
@@ -62,17 +61,13 @@
...
-			emagged = 1
-			src.add_overlay(image('icons/obj/storage.dmi', icon_sparking))
-			sleep(6)
-			src.overlays = null
-			add_overlay(image('icons/obj/storage.dmi', icon_locking))
-			locked = 0
-			user << "<span class='notice'>You short out the lock on [src].</span>"
-
+/obj/item/weapon/storage/secure/attackby(obj/item/weapon/W, mob/user, params)
+	if(istype(W, /obj/item/weapon/card/emag))
```
Comment (JJRcop) [inline code/game/objects/items/weapons/storage/secure.dm]: So this exists only to insult people who try to use the emag

Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/weapons/storage/secure.dm]: yes

**Comment by nullbear** [comment]
There is also a multitool right on the bridge.

**Comment by PKPenguin321** [comment]
supposing that the disk is placed in a secure briefcase that is then moved that is really irrelevant

removing emag vulnerability is a pointless nerf to ops

**Comment by ExcessiveUseOfCobblestone** [comment]
you could just pull the case instead of saying "HOLD UP VALIDHUNTERS, I GOTTA HACK THIS CASE REEEEAALLLL QUICK"

**Comment by AdamElTablawy** [comment]
honestly this is a pointless nerf to the briefcase making it far easier to hack

**Comment by ExcessiveUseOfCobblestone** [comment]
> honestly this is a pointless nerf to the briefcase making it far easier to hack

It's not a nerf, the probability to successfully hack one has been lowered from 40% to 33%.

**Comment by optimumtact** [comment]
I'm convinced

**Pull request merged**